src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -225,64 +228,49 @@ private LogQuery buildQuery(AdminActivityLogPageData data) {
         return query;
     }
     
+    /**
+     * Selects versions for query.
+     * @param versions
+     * @return
```

Comment:
Can be simplified these three lines into one sentence `Returns ... for versions ...`